style.css

```diff
@@ -529,6 +529,7 @@ hr {
   align-items: center;
   justify-content: center;
   margin-bottom: 78px;
+  margin-top: 10px;
 }
 
```

Comment:
- [ ] Kindly use the `flexbox` to align the `form` element with `class="comment_form"`. It is a major design requirement for this milestone. Besides, using Flexbox to lay out contents in a container makes your design responsive across different screen sizes. 👍 💻